[4.3.0] Update VS Code docs (#555)
wso2/docs-mi: en/docs/develop/creating-artifacts/creating-reusable-sequences.md

7. Click **Create**.

<br/>

- The sequence is created in the `<PROJECT_NAME>/src/main/wso2mi/artifacts/sequences` folder. 

> - If you opt for the **Save the sequence in registry** option in the previous step, the sequence is created in the `<PROJECT_NAME>/src/main/wso2mi/resources/` directory according to the values you specified.

Reviewer: ```suggestion
- If you select the **Save the sequence in registry** option in the previous step, the sequence is created in the `<PROJECT_NAME>/src/main/wso2mi/resources/` directory according to the values you specified.
```

> - The created sequence is also available in the **Palette**, under **All Mediators** > **Generic** > **Call Sequence** and ready for use in other meditation workflows.

Reviewer: ```suggestion
- The created sequence is also available in the **Palette**, under **All Mediators** > **Generic** > **Call Sequence**, and is ready for use in other meditation workflows.
```

    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/defined-sequence.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/defined-sequence.png" alt="Sample Sequence" width="80%"></a>

### Create from a sequence mediator

1. Go to the graphical view of your proxy service.

2. Click on the add (**+**) icon to open the **Palette** to access **Mediators**.

    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/edit-proxy.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/edit-proxy.png" alt="Edit Proxy" width="80%"></a>

3. Select **Call Sequence** under **Mediators** > **Generic**.

    <a href="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/call-sequence.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/create-reusable-sequence/call-sequence.png" alt="Call Sequence" width="30%"></a>

> 4. In the **Call Sequence** pane, select the previously-created sequence from the **Referring Sequence** dropdown. Then the sequence will be linked to the integration.

Reviewer: ```suggestion
4. In the **Call Sequence** pane, select the previously created sequence from the **Referring Sequence** dropdown. Then the sequence will be linked to the integration.
```
